inference-engine/include/vpu/vpu_plugin_config.hpp

```diff
@@ -123,11 +123,13 @@ DECLARE_VPU_CONFIG_VALUE(NDHWC);
 DECLARE_VPU_CONFIG_KEY(CUSTOM_LAYERS);
 
 /**
+ * @deprecated IR statistic is not available in IR v10.
  * @brief Ignore statistic in IR by plugin.
  * Plugin could use statistic present in IR in order to try to improve calculations precision.
  * If you don't want statistic to be used enable this option.
  * This option should be used with values: CONFIG_VALUE(YES) or CONFIG_VALUE(NO) (default)
  */
+INFERENCE_ENGINE_DEPRECATED("IR statistic is not available in IR v10")
```

Comment:
I see from changes you removed processing that option. Should we remove option itself then?

Reply:
If some customer still use this option in their apps, you will have compilation error. Our process - deprecate first, live with deprecation at least 2 release, remove then. 

Anyway, even if you are going to remove this option (as VPU plugin), all other option should be removed as well since VPU has a lot of deprecated ones.

Reply:
But following this logic, does it mean if some customer still use this option in their apps we will have runtime error?
Objective about other deprecated options does not work here because for them processing code is not removed..

Reply:
Ok, I see, good catch. Let's discuss offline what we can do with this option.